src/index.html

```diff
@@ -10,32 +10,263 @@
 </head>
 
 <body>
+
   <!--================================= Page Header =================================-->
   <header>
     <div class="header-overlay">
       <div class="header-nav">
         <div class="header-logo"><img src="images/header/hofmann.png" alt="hofmann"></div>
         <nav class="nav-bar">
-          <a class="nav-item-hotels" href="#hotels">Hotels</a>
-          <a class="nav-item-airlines" href="#airlines">Airlines</a>
-          <a class="nav-item-vacations" href="#vacations">Vacations</a>
-          <a class="nav-item-find-more" href="#find-more">Find more</a>
+          <a class="nav-item-hotels text-uppercase" href="#hotels">Hotels</a>
+          <a class="nav-item-airlines text-uppercase" href="#airlines">Airlines</a>
+          <a class="nav-item-vacations text-uppercase" href="#vacations">Vacations</a>
+          <a class="nav-item-find-more text-uppercase" href="#find-more">Find more</a>
         </nav>
         <ul class="header-socials">
           <li class="social-twiter"><a href="#"><img src="images/header/twiter.png" alt="twiter"></a></li>
           <li class="social-facebook"><a href="#"><img src="images/header/facebook.png" alt="facebook"></a></li>
           <li class="social-instagram"><a href="#"><img src="images/header/instagram.png" alt="instagram"></a></li>
         </ul>
       </div>
-    </div>
-    <div class="header-title">
-      <p>See The Unmatched Beauty Of The Great Lakes</h1>
-    </div>
-    <div class="header-video">
+      <div class="header-title title">
+        <p>See The Unmatched Beauty Of The Great Lakes</h1>
+      </div>
+      <div class="header-video">
         <div class="play-background"><img class="play-btn" src="images/header/play.png" alt="play"></div>
+      </div>
+      <div class="header-down-arrow">
+        <img src="images/header/down-arrow.png" alt="down-arrow">
+      </div>
     </div>
   </header>
 
+  <!--================================= Section Feature =================================-->
```

Comment:
Wrap all section in main tag and add style {max-width: 1440px;
  margin: 0 auto;} for it